.github/workflows/github-ci.yml

```diff
@@ -107,24 +107,25 @@ jobs:
 
   build_production:
     # Build Vlasiator witth production flags
-    runs-on: ubuntu-latest
-    container: ursg/vlasiator_ci:20230220_1
+    runs-on: carrington
+      #container: ursg/vlasiator_ci:20230220_1
     needs: build_libraries
 
     steps:
     - name: Checkout source
       uses: actions/checkout@v3 
-    - name: Download libraries
-      uses: actions/download-artifact@v3
-      with:
-        name: libraries
-    - name: Unpack libraries
-      run: tar --zstd -xvf libraries.tar.zstd
-    - uses: ursg/gcc-problem-matcher@master
+      with: 
+        submodules: true
+          #    - name: Download libraries
+          #      uses: actions/download-artifact@v3
+          #      with:
+          #        name: libraries
+          #    - name: Unpack libraries
+          #      run: tar --zstd -xvf libraries.tar.zstd
+          #    - uses: ursg/gcc-problem-matcher@master
     - name: Compile vlasiator (Release build)
       run: |
-        VLASIATOR_ARCH=github_actions make clean
-        VLASIATOR_ARCH=github_actions make -j 3
+        srun -M carrington --job-name tp_compile --interactive --nodes=1 -n 1 -c 16 --mem=40G -p short -t 0:10:0 bash -c 'module purge; module load GCC/11.2.0; module load OpenMPI/4.1.1-GCC-11.2.0 ; module load PMIx/4.1.0-GCCcore-11.2.0; module load PAPI/6.0.0.1-GCCcore-11.2.0; export VLASIATOR_ARCH=carrington_gcc_openmpi; make clean; make -j 16'
```

Comment:
The job naim should probably not be "tp_compile" here, if it is building a release build.